### DISCUSSION TITLE
redirect https://sunshinestream.github.io/ to here

### DISCUSSION THREAD

**Discussion by maxgalloway** [LizardByte/community#57] [Sunshine] [answered]
When you google "sunshine stream" the old page still comes up with a 404. It seems like github pages does not allow for 301 redirects. But you might want to put up a page again with just meta redirect in it.

**Comment by ReenigneArcher** [comment, marked as answer]
We're moving to a different org per https://github.com/SunshineStream/Sunshine/pull/263

I don't know how long this old org will even remain up, and don't intend to waste time and energy on something like this.

With that said. If you were to create a very simple repo with a `gh-pages` branch and a single `index.html` redirecting users to `https://app.lizardbyte.dev` that would be acceptable. I would then import the repo here and it would host on GitHub pages.
